tests/queue/queue_shortcuts_usm_fp64.cpp

```diff
@@ -29,13 +28,10 @@ using namespace queue_shortcuts_usm;
 
 TEST_CASE("queue shortcuts unified shared memory fp64", "[queue]") {
   auto queue = util::get_cts_object::queue();
-  using availability =
-      util::extensions::availability<util::extensions::tag::fp64>;
-  if (!availability::check(queue)) {
-    WARN(
+  if (!queue.get_device().has(sycl::aspect::fp64)) {
+    SKIP(
```

Comment:
Warm -> STOP

Just confirming that  the chang want intended

Reply:
> Just confirming that the change was intended

Yes, it was intended. Note `return;` statement which was removed from the `if`. I expect that the test will now be reported as actually skipped on an unsupported HW instead of reporting itself as passing one.